G2P.pm

```diff
@@ -921,6 +1230,18 @@ sub synonym_mappings {
   $self->{prev_symbol_mappings} = $synonym_mappings;
 }
 
+=head2 write_report
+
+  Arg [1]    : String $flag
+  Arg [2]    : Array of values that need to be written to the log file together with the flag.
+  Example    :
+  Description:
```

Comment:
These are empty. The example is not needed but could have a small description even though is self-explanatory.